external-crates/move/crates/move-analyzer/tests/generate_ide_tests.rs

```diff
@@ -0,0 +1,586 @@
+// Copyright (c) The Move Contributors
```

Comment:
@awelc  I don't think we should land this file. We should use it to convert the tests and then delete it.

Reply:
Agreed!